src/envelope.js

```diff
@@ -902,4 +902,12 @@ define(function (require) {
     }
   };
 
+  // Different name for backwards compatibility, replicates p5.Envelope class
+  p5.Env = function(t1, l1, t2, l2, t3, l3) {
+    console.warn('WARNING: p5.Env is now deprecated and may be removed in future versions. ' +
```

Comment:
If you'd be up for a quick find-and-replace-all, especially in the `src` and `examples` directories, that'll be helpful to complete the deprecation.

You could do it in one line with a recursive `grep` for `p5.Env(`, piped into `sed`

```
grep -rl "p5.Env(" examples src | xargs sed -i '' 's/p5.Env(/p5.Envelope(/g'
```

...but you'd need to preserve the two `p5.Env`'s in the two lines above